editor/filesystem_dock.cpp

```diff
@@ -2516,6 +2547,35 @@ void FileSystemDock::_feature_profile_changed() {
 	_update_display_mode(true);
 }
 
+void FileSystemDock::set_file_sort(FileSortOption p_file_sort) {
+	for (int i = 0; i != FILE_SORT_MAX; i++) {
+		tree_button_sort->get_popup()->set_item_checked(i, (i == (int)p_file_sort));
+		file_list_button_sort->get_popup()->set_item_checked(i, (i == (int)p_file_sort));
+	}
+	file_sort = p_file_sort;
+
+	// Update everything needed
+	_update_tree(_compute_uncollapsed_paths());
+	_update_file_list(true);
+}
+
+void FileSystemDock::_file_sort_popup(int p_id) {
+	set_file_sort((FileSortOption)p_id);
+}
+
+MenuButton *FileSystemDock::_create_file_menu_button() {
+	MenuButton *button = memnew(MenuButton);
+	button->set_flat(true);
+	button->set_tooltip(TTR("Sort files"));
+
+	PopupMenu *p = button->get_popup();
+	p->connect("id_pressed", callable_mp(this, &FileSystemDock::_file_sort_popup));
+	p->add_radio_check_item("Sort by name", FILE_SORT_NAME);
+	p->add_radio_check_item("Sort by type", FILE_SORT_TYPE);
```

Comment:
```suggestion
	p->add_radio_check_item("Sort by Name", FILE_SORT_NAME);
	p->add_radio_check_item("Sort by Type", FILE_SORT_TYPE);
```

Reply:
Done :)

Reply:
@groud I believe you have to re-request review or dismiss current review.